bin/setup-docker.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# This setup script is used with the docker-ckan dev stack
+
+echo '===== Setup datagovuk publish ====='
+
+RETRIES=10
+until psql -h db -U ckan -d ckan -c "select 1" > /dev/null 2>&1 || [ $RETRIES -eq 0 ]; do
+    echo "Waiting for postgres server, $((RETRIES--)) remaining attempts..."
+    sleep 1
+done
+
+bin/rails db:setup
+rails db:environment:set RAILS_ENV=docker
```

Comment:
I think we should avoid adding a new environment for Docker like this as it means we need to maintain an extra file in `config/environments` and it would also be quite surprising for new developers I think as usually there's only a `development`, `production` and `test` environment.

For the things we need to customise like the elasticsearch host and `ckan_v26_base_url` could we use environment variables?

Reply:
hmmm, ok, I'll see if I can set up env vars instead